### ISSUE TITLE
Progress Bar Navigation Enhancements

### ISSUE THREAD

**Issue by tfoxley** [WCF-Insurance/WCF-Cashmere#303]
![image-2020-09-15-02-17-17-470](https://user-images.githubusercontent.com/344920/103685711-e8fcfc00-4f4a-11eb-80aa-917a6ad89cac.png)

The following styling and functionality changes need to be built into the progress bar:
 * Link that can be visited - currently working as it should (I believe the text is black) no changes needed.
 * Link that cannot be visited - the lock and lighter gray colored text
 * Link that has been "completed" (and can be visited - a future page that cannot be visited must also not be completed) - currently working as desired

In addition, we need to fix the following in the progress bar:
* Remove the hard media query that makes the progress bar "display: none" at 1024px. This should be handled via the breakpoint attribute.
* On the "mobile-progress-items" element, remove the margin-left on the first button and the margin-right on the last button so that the progress bar sits flush with the container it is nested in. (this is referring to the navigation arrows on the sides of the progress items)

To Summarize:
* Add functionality for locked link that cannot be visited
* Media query change
* Margin on the navigation arrows in mobile view
